tests/stdlib/tsums.nim

```diff
@@ -0,0 +1,22 @@
+import std/sums
+from math import pow
+
+var epsilon = 1.0
+while 1.0 + epsilon != 1.0:
+  epsilon /= 2.0
+let data = @[1.0, epsilon, -epsilon]
+assert sumKbn(data) == 1.0
+assert sumPairs(data) != 1.0 # known to fail
```

Comment:
```suggestion
# assert sumPairs(data) != 1.0 # known to fail
```

Reply:
It is disabled now.

Reply:
Use when (likewise, when defined posix is used a lit in runnableExamples)